packages/forms/test/form_group_spec.ts

```diff
@@ -148,6 +154,56 @@ describe('FormGroup', () => {
       expect(g.value).toEqual({'one': '1'});
       expect(g.valid).toBe(true);
     });
+
+    it('should not emit events when `FormGroup.addControl` is called with `emitEvent: false`',
+       () => {
+         const g = new FormGroup({'one': new FormControl('1')});
+         expect(g.value).toEqual({'one': '1'});
+         expect(g.valid).toBe(true);
+
+         g.valueChanges.subscribe(() => logger.push('value change'));
+         g.statusChanges.subscribe(() => logger.push('status change'));
+
+         g.addControl('two', new FormControl('2', Validators.minLength(10)), {emitEvent: false});
```

Comment:
I think having a `minLength` validator in this test is not required, so we can remove it:
```suggestion
         g.addControl('two', new FormControl('2'), {emitEvent: false});
```

Reply:
Ah OK, I had added that to check that `statusChanges` did not emit when the status changed (in this case `VALID` -> `INVALID`), but it seems that `statusChanges` emits every time it recalculates, not just when there is a change, so you are right and this validator is unnecessary.